### ISSUE TITLE
Look into what should be done with audio and video on Fish Bowl

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#310]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/fishbowlie/

Chrome dev tools are complaining the video should have a label (and tracks) and audio should have controls.

Likely the video is presentational and we can just hide it (check to see if aria-hidden fixes this).
Audio likely can be fixed same as other demos with pause button.

![screen shot 2016-12-30 at 4 10 45 am](https://cloud.githubusercontent.com/assets/465069/21559546/fb4536fa-ce45-11e6-916e-575775381ac3.png)

**Comment by dstorey** [comment]
Can confirm aria-hidden on video fixes that issue.
